deps/spidershim/src/conversions.h

```diff
@@ -61,6 +61,11 @@ static inline JSObject* GetObject(Value* val) {
   return v->isObject() ? &v->toObject() : nullptr;
 }
 
+static inline JSObject* GetObject(Template* val) {
+  // XXXbz Can we assume val is never null here?  Seems dubious.
```

Comment:
Most of these helpers assume their argument is non-null, and leave it up to the caller to ensure that.  I'd be fine with assuming the same here.